include/gz/sensors/CameraSensor.hh

```diff
@@ -175,6 +176,10 @@ namespace gz
       /// \param[in] _scene Pointer to the new scene.
       private: void OnSceneChange(gz::rendering::ScenePtr /*_scene*/);
 
+      /// \brief Callback for triggered subscription
+      /// \param[in] _msg Boolean message
+      private: virtual void OnTrigger(const gz::msgs::Boolean &/*_msg*/);
```

Comment:
looks like this changed the v-table layout maybe ABI-incompatible. We'll need to make this function non-virtual in `gz-sensors7`. This can be made virtual later when we forward port to `main` where we can break ABI.

Reply:
Done in [b7ecaa5](https://github.com/gazebosim/gz-sensors/pull/322/commits/b7ecaa5922c4baac383a6ae09114a8ef0fe66ff9)